.github/workflows/IJ.yml

```
@@ -17,9 +17,9 @@ jobs:
         IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.3, IU-2024.1]
```

just spotted that we're missing IJ-2024.2 here .
```suggestion
        IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.3, IU-2024.1, IU-2024.2]
```